lms/djangoapps/teams/views.py

```diff
@@ -1407,7 +1407,7 @@ def post(self, request, **_kwargs):
         team_import_manager.set_team_membership_from_csv(inputfile_handle)
 
         if team_import_manager.import_succeeded:
-            msg = "Successfully added {} students to teams".format(team_import_manager.number_of_records_added)
```

Comment:
Are we using `number_of_records_added` anywhere besides this status message?  If not, let's delete it?

Reply:
Removing. I am guilty of keeping things "just in case" :)